### PR TITLE
Add simple router for admin

### PR DESCRIPTION
**Pull request by solopiy**

### CODE REVIEW

**Comment by solopiy** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;295p7h9R&#x2F;315-when-i-m-in-admin-panel-i-want-to-see-link-to-list-of-posts-so-i-can-open-it"> When I&#39;m in admin panel I want to see link to list of posts so I can open it</a>

File: app/concepts/admin/index.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Router, Route, browserHistory } from 'react-router';
+import { render } from 'react-dom';
+
+import StartPage from './containers/start_page/start_page.jsx';
```
Comment (ezhlobo) [inline app/concepts/admin/index.jsx]: What do you think about calling it `./containers/StartPage/StartPage.jsx`?

Actually I want to discuss it. //CC @solopiy @Proghat

File: app/concepts/admin/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import PostsList from './containers/posts_list/posts_list.jsx';
+
+render(
+  <Router history={browserHistory}>
+    <Route>
```
Comment (ezhlobo) [inline app/concepts/admin/index.jsx]: Empty route but I don't suggest you to delete it. What do you think about wrapping all your routes into `/admin` scope?

File: app/concepts/admin/containers/StartPage/StartPage.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+class StartPage extends React.Component {
+    render() {
+        return (
+          <div>
+            <Link to="admin/posts">List of posts</Link>
```
Comment (ezhlobo) [inline app/concepts/admin/containers/StartPage/StartPage.jsx]: Let's start all links with `/` as you did in PostsList

File: config/assets_builder.js [hunk 4]
```diff
@@ -67,6 +67,14 @@ module.exports = {
                 include: path.resolve(__dirname, '../app/'),
                 exclude: /node_modules/,
             },
+            {
+                test: /.jsx$/,
```
Comment (ezhlobo) [inline config/assets_builder.js]: I still don't understand why you add the same loaders for `.jsx` twice

**Pull request merged**